### ISSUE TITLE
InteroType strips away last char

### ISSUE THREAD

**Issue by DavsX** [myfreeweb/intero.nvim#2]
When I select a function/code manually and call InteroType, the last letter of the selection is removed. The issue is in  [here](https://github.com/myfreeweb/intero.nvim/blob/master/autoload/intero/util.vim#L6) with the '- 2' at the end. There is an updated version of this code snippet here: http://stackoverflow.com/a/6271254

In the current version to make it behave correctly I need to select the function and the next char.

**Comment by valpackett** [comment]
Yeah, that was intended, but probably wrong... I'll fix this soon

**Comment by Xandaros** [comment]
Ohhh, so that's why goto und uses weren't working.
I do wonder, though - what made you decide to do it like this? :D

If you can, it would be great if goto and uses worked even if you just hover over the word, but I suppose I can just make my binding `viw:InteroGoto<CR>` or something.
For the time being, I guess I'll just add an `l` in there.
